src/marqo/tensor_search/api.py

```diff
@@ -123,31 +123,9 @@ def marqo_api_exception_handler(request: Request, exc: api_exceptions.MarqoWebEr
         return JSONResponse(content=body, status_code=exc.status_code)
 
 
-@app.exception_handler(pydantic.ValidationError)
-async def validation_exception_handler(request: Request, exc: pydantic.ValidationError) -> JSONResponse:
```

Comment:
Is this being removed because the handler is obsolete?

Reply:
Got this. In the future then, we may need to make a handler to handle `fastapi.exceptions.RequestValidationError`, in case the default FastAPI handler isn't sufficient? It might be worth making a ticket for when the need arises.